recipes/default.rb

```diff
@@ -1,13 +1,79 @@
-package value_for_platform(
-  ['centos', 'redhat'] => { 'default' => 'tor-core' },
-  ['debian', 'ubuntu'] => { 'default' => 'tor' }
-)
+#
+# Author:: Richard Klafter (<rpklafter@yahoo.com>)
+# Cookbook Name:: tor
+# Recipe:: default
+#
 
+case node['platform_family']
+# Debian / Ubuntu
+when 'debian'
+  include_recipe 'apt'
+
+  # Add TorProject.org repository
+  apt_repository 'tor' do
+    uri          'http://deb.torproject.org/torproject.org'
+    distribution node['lsb']['codename']
+    components   ['main']
+    keyserver    'keys.gnupg.net'
+    key          '886DDD89'
```

Comment:
Use the full fingerprint... not just the 32bit key id. For more info: https://evil32.com/